tractor/_streaming.py

```diff
@@ -396,6 +398,10 @@ async def cancel(self) -> None:
                     log.warning(
                         "May have failed to cancel remote task "
                         f"{cid} for {self._portal.channel.uid}")
+                else:
+                    log.warning(
```

Comment:
This is just new logging, yes noisy but handy until this cancel stuff is better refined imo.

We can decrease the level if complaints pile in.